sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveExternalCatalog.scala

```diff
@@ -93,8 +92,11 @@ private[spark] class HiveExternalCatalog(conf: SparkConf, hadoopConf: Configurat
     }
   }
 
-  private def requireTableExists(db: String, table: String): Unit = {
-    withClient { getTable(db, table) }
+  /**
+   * Get the raw table metadata from hive metastore directly.
+   */
+  private def getRawTable(db: String, table: String): CatalogTable = withClient {
```

Comment:
Is it worthy to create a new method instead of using `client.getTable(db, table)` directly?